components/popover/index.tsx

```diff
@@ -11,7 +11,7 @@ export interface PopoverProps extends AbstractTooltipProps {
 export default class Popover extends React.Component<PopoverProps, {}> {
   static defaultProps = {
     prefixCls: 'ant-popover',
-    placement: 'top',
+    placement: 'top' as 'TooltipPlacement',
```

Comment:
`TooltipPlacement` is a type alias so quotes are not needed.
Should be:
```
placement: 'top' as TooltipPlacement,
```

Reply:
I am getting error, Cannot find TooltipPlacement

Reply:
@GauravV-02 You probably have to import that type from https://github.com/ant-design/ant-design/blob/7fa05996957ef0eded21d810d71364d294ea947d/components/tooltip/index.tsx#L10 

So...
```
import { TooltipPlacement } from '...filePath...'
```

Reply:
I tried importing it before got error,  but this time it worked